README.md

```diff
@@ -1,25 +1,38 @@
 # dp-search-api
 
-Digital Publishing Search API
+Dissemination Service Search API
 
 A Go application microservice to provide query functionality on the ONS Website
 
 ## Getting started
 
 Set up dependencies locally as follows:
 
-* In [dp-compose](https://github.com/ONSdigital/dp-compose) run `docker-compose up -d` to run ElasticSearch 7.10
-    * dp-compose will run Elasticsearch 7.10 on port 11200 to not conflict with ES 2.2 running on port 9200
-* If using the POST /search endpoint then authorisation for this requires running Vault and Zebedee as follows:
-* In any directory run `vault server -dev` as Zebedee has a dependency on Vault
-* In the zebedee directory run `./run.sh` to run Zebedee
+* See our [Docker Compose Search Stack](https://github.com/ONSdigital/dp-compose/tree/main/v2/stacks/search) to run all services end to end.
 
-Then run `make debug`
+Alternatively you can run individual services:
+
+In order to run the dp-search-api and surface results you will also need to populate the index in some way. This can be done using one of the following:
+
+* [zebedee](https://github.com/ONSdigital/zebedee) with local content generated by [dp-zebedee-content](https://github.com/ONSdigital/dp-zebedee-content#dp-zebedee-content)
+* [dp-dataset-api](https://github.com/ONSdigital/dp-dataset-api)
```

Comment:
To explain where the local content for populating the index comes from if using the dataset api, rather than zebedee, I would add this:
```suggestion
* [dp-dataset-api](https://github.com/ONSdigital/dp-dataset-api) with content in a local MongoDB, which could be created manually or port-forwarded to.
```